### PR TITLE
Add point polling part 1

### PR DESCRIPTION
**Pull request by Kexort**

### CODE REVIEW

File: src/MainRouter.tsx [hunk 1]
```diff
@@ -105,6 +107,15 @@ const MainRouter = (): ReactElement => {
     }
   };
 
+  const getPoints = async () => {
+    fetch(process.env.BACKEND_API_URL + "/points/" + username).then(
```
Comment (bosi95) [inline src/MainRouter.tsx:111]: missing catch for error handling

File: src/pages/ClaimRewardPage/ClaimRevardPage.tsx [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
 import { ROUTES } from "../../utils/constants";
 
 const ClaimRewardPage: React.FC = () => {
+  const { username } = useGlobalState();
   const navigate = useNavigate();
   const inputRef = useRef<HTMLInputElement>(null);
 
+  useEffect(() => {
+    const nonce = "nonce";
+    fetch(process.env.BACKEND_API_URL + "/nonce/" + username).then((resp) =>
```
Comment (bosi95) [inline src/pages/ClaimRewardPage/ClaimRevardPage.tsx:17]: missing catch for error handling

File: src/pages/ClaimRewardPage/ClaimRevardPage.tsx [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
+      resp.text().then((data) => {
+        console.log("nonce fetched", data);
+        //TODO: sign nonce with private key
+      })
+    );
+    fetch(process.env.BACKEND_API_URL + "/redeem", {
```
Comment (bosi95) [inline src/pages/ClaimRewardPage/ClaimRevardPage.tsx:23]: missing catch for error handling

**Review by bosi95 — APPROVED** [review]
approved, if error handling is fixed

File: src/pages/ClaimRewardPage/ClaimRevardPage.tsx (hunk 3 above)
Comment (bosi95) [reply src/pages/ClaimRewardPage/ClaimRevardPage.tsx]: fixed

File: src/pages/ClaimRewardPage/ClaimRevardPage.tsx (hunk 2 above)
Comment (bosi95) [reply src/pages/ClaimRewardPage/ClaimRevardPage.tsx]: fixed

File: src/MainRouter.tsx (hunk 1 above)
Comment (bosi95) [reply src/MainRouter.tsx]: fixed

**Review by aranyia — APPROVED** [review]

**Pull request merged**